docs/en/latest/installation-guide.md

```diff
@@ -271,9 +272,11 @@ You can configure your APISIX deployment in two ways:
    apisix start -c <path to config file>
    ```
 
-APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured.
+APISIX will use the configurations added in this configuration file and will fall back to the default configuration if anything is not configured. Generally, APISIX gets installed at `/usr/local/apisix/` directory, so your configuration file will be present at `/usr/local/apisix/conf/` path.
+
+In case you get the Port binding logs when trying to run the APISIX server using `apisix start` command, it is most likely that the ports are being used by certain processes running on your machine. Try deleting the process using the port or configure the default listening port to other available port on your local machine.
 
-For example, to configure the default listening port to be `8000` without changing other configurations, your configuration file could look like this:
+In order to configure the default listening port to be `8000` without changing other configurations, your configuration file could look like this:
```

Comment:
Why the change here? `For example` + `could` sound like a pair providing a sample config, but now the instruction sounds definitive, which it should not be.

Reply:
If you read the sentence above it that mentions `configure the default listening port to other available port on your local machine.`... this makes sense. We're showing how do you perform the last step mentioned in the documentation rather than giving you an example which IMO is vague at this point.

Reply:
then `could` --> `should`

```suggestion
In order to configure the default listening port to be `8000` without changing other configurations, your configuration file should look like this:
```